docs/theming/high-contrast-mode.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Hide Contrast Mode
+initialTab: 'preview'
+inlineHtmlPreviews: true
```

Comment:
What does `initialTab` and `inlineHtmlPreviews` do? I couldn't find anything in Docusaurus or within the docs source code.

Reply:
https://github.com/ionic-team/ionic-docs/pull/3431/commits/bcafcbe89d5614d96dab5dcfa93b6785fb98e359